server/controllers/stock/reports/stock_exit_service.receipt.handlebars

```diff
@@ -14,23 +15,26 @@
           <span class="text-capitalize">{{translate 'FORM.LABELS.EMAIL'}}</span>: {{enterprise.email}}
         </p>
       </div>
+      
       <div class="col-xs-6 text-right">
         <h3 style="margin: 0px;">
           <span class="text-uppercase">
             {{translate 'STOCK.RECEIPT.EXIT_SERVICE'}}
           </span> <br>
           <strong>{{details.service_display_name}}</strong> <br>
-          <span>{{details.document_reference}}</span> <br>
-          <small>{{date details.date}}</small> <br>
-          <br>
+          <div class="row">
+            <small>{{> barcode value=details.barcode}}</small>
+            <br/><br/>
```

Comment:
Could you remove this `<br/><br/>` and add one `<br>` after the `div` block before the `</h3>` tag